cmd/cfssl/testdata/config.json

```diff
@@ -0,0 +1,20 @@
+{
+    "signing": {
+        "default": {
+            "usages": [
+                "cert sign"
+            ],
+            "expiry": "720h",
+            "auth_key": "garbage"
+        }
+    },
+    "auth_keys": {
+        "garbage": {
+            "type": "standard",
+            "key": "f66761a3098baaa7c893412d54557b8ff22e69eb4f26f958b769ff6ff7f7271b"
+        }
+    },
+    "remotes": {
+        "localhost": "127.0.0.1:8888"
+    }
+}
```

Comment:
<img width="166" alt="screenshot 2015-12-03 16 15 00" src="https://cloud.githubusercontent.com/assets/3589480/11578307/129eef6e-99d9-11e5-889b-f7cdc0620167.png">

I believe it's common practice to have a newline (\n) at the end of files